src/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { useState, HTMLAttributes } from 'react';
+
+export interface ContextProps {
+  selected: string,
+  setSelected: (value: string) => void
+}
+export const TabContext = React.createContext<ContextProps>({
+  selected: '',
+  setSelected: () => { }
+});
+
+type Props = HTMLAttributes<HTMLDivElement>
+
+const Tabs: React.FC<Props> = ({ children }) => {
+  const [selected, setSelected] = useState('');
+  const value = { selected, setSelected };
+
+  return (
+    <TabContext.Provider value={value}>
```

Comment:
Seems a bit redundant to set `value` if it's just passing along the useState vars. Perhaps just `value={ selected, setSelected }` would work?

This isn't a big thing really, just tidiness.

Reply:
Done